book_source/source/module-4-libraries-for-data-science/lesson-11-machine-learning/introduction.md

```diff
@@ -83,7 +81,7 @@ You might have noticed from the last paragraph, that we said you need to give it
 
 A **machine learning algorithm** (commonly, just called a "learning algorithm") takes this training set and outputs a **model** that it thinks will do the best at the task. A **model** is the "rules" to predict the value for any given example.  
 
-For most setups in machine learning, this training set needs to have **features** and **labels** . The **labels** are the correct values for an example in the training set and the **features** are the description of the example itself. Notice this definition of features is awfully vague! It turns out one of the hardest parts of machine learning is coming up with a good set of features to describe your data that is useful for the learning algorithm to learn from!  
+For most setups in machine learning, this training set needs to have **features** and **labels**. The **labels** are the correct values for an example in the training set and the **features** are the description of the example itself. Notice this definition of features is awfully vague! It turns out one of the hardest parts of machine learning is coming up with a good set of features to describe your data that is useful for the learning algorithm to learn from!  
```

Comment:
Can't comment about this in the right spot, but the "ML Definition" quote absorbed the next paragraph of text. The blockqutoe should just be the first sentence.

I don't know why the translation script did this, but blockquotes are generally pretty sus from my experience.

Reply:
Thanks for catching! I think it just need one more extra blank line for it to recognize the next paragraph is using a different format.